index.d.ts

```diff
@@ -0,0 +1,16 @@
+export = attacher;
```

Comment:
Could _index.d.ts_ be added to the gitignore?
Since it is a generated file, would like to avoid people trying to edit it, only for the edits to be overwritten by the build.